pallets/moonbeam-foreign-assets/src/benchmarks.rs

```diff
@@ -72,7 +72,7 @@ mod benchmarks {
 		)?;
 
 		// Remove ethereum receipts
-		pallet_ethereum::Pending::<T>::kill();
+		let _ = pallet_ethereum::Pending::<T>::clear(pallet_ethereum::Pending::<T>::count(), None);
```

Comment:
The idea is to get rid of this line. The temporary fix I added in the previous release was not correct, it hides the side-effect the `Pending` storage has on ethereum transaction costs.

Reply:
To give some more context, this was necessary because in the benchmark `create_foreign_asset` we were registering a total of `MaxForeignAssets - 1` in the setup and it was resulting in > 3MB of proof_size.